contents/tutorials/multiple-environments.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to setup PostHog for multiple environments (development, staging, production)
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2023-01-02
+featuredImage: ../images/tutorials/banners/multiple-environments.png
+topics: ["configuration"]
+---
+
+Many software development teams use environments to split up their code, such as development, staging, and production. This ensures changes in development don’t affect production, helps teams test before release, and increases the quality of code that reaches end users.
+
+Using multiple environments requires splitting the data from each of them. If not, development and staging data combine with production data and can cause inaccuracy and issues. This tutorial goes over how to prevent those issues by setting up PostHog for use in multiple environments.
+
+## Using multiple projects
+
+The best practice for using PostHog with multiple environments is using multiple projects. This enables developers to test event capture, session recordings, and feature flags without polluting the data in production.
+
+Creating a new project in PostHog is simple. In your instance, click your project name and then the “New project” button. Be sure to name your project to make it clear it is not production, like “development” or “staging.”
+
+![Projects](../images/tutorials/multiple-environments/project.png)
+
+This takes you through the project setup flow again and gives you a new project API key. You can keep your old key for production, and add the new one in the relevant environment. You'll likely need environment variables for this (like a `.env` file) if you haven't set those up. Initializing PostHog with a different key in each environment connects them to a different project.
```

Comment:
Should we cover how to set up env variables here? The phrasing suggests I should expect it to be covered, but it's a bit out of my technical understanding

Reply:
Added it